src/gif.rs

```diff
@@ -170,7 +171,7 @@ impl<R: Read> Iterator for GifFrameIterator<R> {
                     f_height = u32::from(frame.height);
 
                     // frame.delay is in units of 10ms so frame.delay*10 is in ms
-                    delay = Ratio::new(frame.delay * 10, 1);
+                    delay = Ratio::new(frame.delay as u32 * 10, 1);
```

Comment:
This should be `u32::from(frame.delay)` to make it more obvious this is lossless.